base/strings/basic.jl

```diff
@@ -394,27 +396,41 @@ end
 """
     prevind(str::AbstractString, i::Integer, n::Integer=1) -> Int
 
+Case `n == 1`.
```

Comment:
Perhaps make this a bullet point list instead?

Reply:
fixed